### PR TITLE
Changed to display recent photos first.

### PR DESCRIPTION
**Pull request by a2444mo10**
Changed to display recent photos first.
The screen transitions have also been changed.
Target version has been changed to iOS13 or higher.

### CODE REVIEW

**Comment by tyokujin** [comment]
@a2444mo10 
Could you resolve conflict on base branch?

**Comment by a2444mo10** [comment]
Conflicts have been resolved.

File: NohanaImagePicker/AlbumListViewController.swift [hunk 1]
```diff
@@ -37,14 +43,14 @@ class AlbumListViewController: UITableViewController, EmptyIndicatable, Activity
...
     weak var nohanaImagePickerController: NohanaImagePickerController?
     var photoKitAlbumList: PhotoKitAlbumList!
+    weak var delegate: AlbumListViewControllerDelegate?
 
     override func viewDidLoad() {
         super.viewDidLoad()
         if let nohanaImagePickerController = nohanaImagePickerController {
             view.backgroundColor = nohanaImagePickerController.config.color.background ?? .white
             title = nohanaImagePickerController.config.strings.albumListTitle ?? NSLocalizedString("albumlist.title", tableName: "NohanaImagePicker", bundle: nohanaImagePickerController.assetBundle, comment: "")
-            setUpToolbarItems()
```
Comment (tyokujin) [inline NohanaImagePicker/AlbumListViewController.swift:46]: Could you remove unused function.

File: NohanaImagePicker/AlbumListViewController.swift [hunk 2]
```diff
@@ -17,6 +17,12 @@
 import UIKit
 import Photos
 
+protocol AlbumListViewControllerDelegate: AnyObject {
+    func didSelectMoment()
+    func didSelectAlbum(album: PhotoKitAssetList)
+    func dissmissWithoutDoingAnything()
```
Comment (tyokujin) [inline NohanaImagePicker/AlbumListViewController.swift:23]: Why do you name `dismissWithoutDoingAnything `?
I think when run dismiss on delegate, calling function is do action. 
So, I feel unnatural naming.   
What about `dismiss`?

**Comment by tyokujin** [comment]
Please change `UIActivityIndicatorView(style: .medium)` from `UIActivityIndicatorView(style: .gray)`.
https://github.com/nohana/NohanaImagePicker/blob/e494913f22e4eaf4e5b84d6ccc906e4dd7f9936c/NohanaImagePicker/AlbumListViewController.swift#L222

File: NohanaImagePicker/AlbumListViewController.swift (hunk 1 above)
Comment (a2444mo10) [reply NohanaImagePicker/AlbumListViewController.swift:46]: This method is implemented in UIViewController extension, so I want to leave it as it is.

File: NohanaImagePicker/AlbumListViewController.swift (hunk 2 above)
Comment (a2444mo10) [reply NohanaImagePicker/AlbumListViewController.swift:23]: Fix it to "dismiss".

**Comment by a2444mo10** [comment]
sorry.
It was supposed that the warning would be resolved by another pull request.
Corrected the content of the review and resolved the warning.

**Comment by a2444mo10** [comment]
I change the base branch and try the pull request again.
Change the target version with another pull request.

**Pull request closed without merging**